integr8ly/integreatly-operator
# MGDAPI-5996 Added oc into external test image

# Issue link


https://issues.redhat.com/browse/MGDAPI-5996

# What


Image for running the test externally (not on the cluster directly) was missing oc.

# Verification steps




Have a cluster with RHOAM installed. Build and push the image from this PR and run some tests using the external image:
ORG=yourQuayOrg make image/external/build
ORG=yourQuayOrg make image/external/push
podman run -it -e OPENSHIFT_HOST='https://api.test-237-tr1457.mf29.s1.devshift.org:6443' -e OPENSHIFT_PASSWORD='redacted' -e MULTIAZ=false -e DESTRUCTIVE=false -e NUMBER_OF_TENANTS='2' -e TENANTS_CREATION_TIMEOUT='3' -e RegExpFilter='A01' -e OUTPUT_DIR=a01-test-run-result -v "$(pwd)/a01-test-run-result:/a01-test-run-result:Z" quay.io/yourQuayOrg/integreatly-operator-test-external:latest

I did all this (ran just A01 test) so eye review might suffice:

```
podman run -it -e OPENSHIFT_HOST='https://api.test-237-tr1457.mf29.s1.devshift.org:6443' -e OPENSHIFT_PASSWORD='redacted' -e MULTIAZ=false -e DESTRUCTIVE=false -e NUMBER_OF_TENANTS='2' -e TENANTS_CREATION_TIMEOUT='3' -e RegExpFilter='A01' -e OUTPUT_DIR=a01-test-run-result -v "$(pwd)/a01-test-run-result:/a01-test-run-result:Z" quay.io/trepel/integreatly-operator-test-external:latest
Login successful.

You have access to 115 projects, the list has been suppressed. You can list all projects with 'oc projects'

Using project "default".
Welcome! See 'oc help' to get started.
INFO[0000] calling reset on all prometheus gauge vectors  custom_metrics=StartGaugeVector
=== RUN   TestAPIs
Running Suite: Functional Test Suite - /
========================================
Random Seed: 1692777634

Will run 1 of 60 specs
------------------------------
[BeforeSuite] 
/go/src/github.com/integr8ly/integreatly-operator/test/functional/suite_test.go:78
  STEP: bootstrapping test environment @ 08/23/23 08:00:41.484
[BeforeSuite] PASSED [0.010 seconds]
------------------------------
SSSSSSS
------------------------------
integreatly managed-api HAPPY PATH A01 - Verify that all stages in the integreatly-operator CR report completed
/go/src/github.com/integr8ly/integreatly-operator/test/functional/integreatly_test.go:148
• [5.516 seconds]
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
[AfterSuite] 
/go/src/github.com/integr8ly/integreatly-operator/test/functional/suite_test.go:107
  STEP: tearing down the test environment @ 08/23/23 08:00:47.012
[AfterSuite] PASSED [0.001 seconds]
------------------------------
[ReportAfterSuite] Autogenerated ReportAfterSuite for --junit-report
autogenerated by Ginkgo
[ReportAfterSuite] PASSED [0.004 seconds]
------------------------------

Ran 1 of 60 Specs in 5.530 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 59 Skipped
```
